cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -132,6 +132,9 @@ public TestCommand() {
     @CommandLine.Option(names = "--debug", description = "start in remote debugging mode")
     private String debugPort;
 
+    @CommandLine.Option(names = "--workers", description = "maximum number of parallel test jobs")
+    private int workers;
```

Comment:
Why do we need to allow the user to configure this?

Did you also consider naming this strands instead of workers?

Reply:
How did we decide to use strands instead of threads?

Reply:
Configuring is allowed because the developer should be able to control the worker count for resource-intensive tests to avoid issues like OOM. Another reason is that we have set the default worker count to 1 (for now) to avoid breaking existing pipelines.